src/python/pants/backend/python/rules/python_test_runner.py

```diff
@@ -19,8 +19,15 @@
 from pants.util.strutil import create_path_env_var
 
 
-@rule(TestResult, [PythonTestsAdaptor, PyTest, PythonSetup, SubprocessEncodingEnvironment])
-def run_python_test(test_target, pytest, python_setup, subprocess_encoding_environment):
+# TODO(7697): Use a dedicated rule for removing the source root prefix, so that this rule
+# does not have to depend on SourceRootConfig.
```

Comment:
This comment looks... resurrected from somewhere, maybe?

Reply:
Yeah. Fixed.